.travis.yml

```diff
@@ -7,14 +7,22 @@ python:
     - 3.6
     - pypy
     - pypy3
+matrix:
+    include:
+        - python: "3.7"
+          dist: xenial
+          sudo: true
 install:
-    - pip install zc.buildout
-    - buildout
+    - pip install zope.testrunner coverage coveralls
+    - pip install -U -e .[test,docs]
 script:
-    - bin/test
-    - cd docs; make SPHINXBUILD=../bin/sphinx-build doctest
+    - coverage run -m zope.testrunner --test-path=src -v
+    - cd docs; make doctest
 notifications:
     email: false
+after_success:
+    - coveralls
```

Comment:
The coverage report [isn't finding any source](https://coveralls.io/builds/19443034). Perhaps we need to add the appropriate `[run] source = transaction` block to `.coveragerc`.

Reply:
I think that's because no tests were run and so absolutely no source files were covered.

Adding a `source = transaction` would explicitly show 0% coverage instead of ?% coverage, which would make it clearer.